src/UserCom.Client/UserComClient.cs

```diff
@@ -19,14 +20,28 @@ public partial class UserComClient : HttpRestClient, IUserComClient
     {
         private readonly ILogger<UserComClient> _logger;
 
-        public UserComClient(UserComAuthenticator userComAuthenticator, ILogger<UserComClient> logger) : base(CreateClient(userComAuthenticator), SerializerSettings, logger)
+        public UserComClient(UserComAuthenticator userComAuthenticator, ILogger<UserComClient> logger) : base(CreateClient(userComAuthenticator, logger), SerializerSettings, logger)
         {
             _logger = logger ?? throw new ArgumentNullException(nameof(logger));
         }
 
-        private static HttpClient CreateClient(UserComAuthenticator authenticator)
+        private static HttpClient CreateClient(UserComAuthenticator authenticator, ILogger<UserComClient> logger)
         {
-            return new HttpClient(authenticator) { BaseAddress = authenticator.ServiceUri };
+            var client = new HttpClient(authenticator) { BaseAddress = authenticator.ServiceUri };
+            
+            if (client != null && logger != null)
+            {
+                if (client.DefaultRequestHeaders != null)
+                {
+                    logger.LogInformation(new EventId(100), $"Authorization header is: {client.DefaultRequestHeaders.Authorization} for {authenticator.ServiceUri}");
+                }
+                else
+                {
+                    logger.LogInformation(new EventId(101), $"Request headers are empty for: {authenticator.ServiceUri}");
+                }                
+            }
+
+            return client;
```

Comment:
Is not client.DefaultRequestHeaders always empty? since httpClient is just initialized? or I am missing something.